scripts/sync/pact_plugins.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require_relative 'support'
+
+SOURCE_REPO = 'pact-foundation/pact-plugins'
+DESTINATION_DIR = relative_path_to('docs/implementation_guides/pact_plugins')
+TRANSFORM_PATH = lambda { |path|
+  File.join(DESTINATION_DIR, path.downcase)
+}
+INCLUDE = [
+  ->(path) { path.end_with?('.md') }
+]
+IGNORE = [
+  ->(path) { path.include?('vendor/') },
+  ->(path) { path.include?('plugin-driver-design.md') }
```

Comment:
I don't want to exclude this file but I think the parentheses around these headers causes an issue, can probably transform it.

https://github.com/pact-foundation/pact-plugins/blob/main/docs/plugin-driver-design.md#content-matcher-content-matcher

Just a thought, will probably need to sync the couple of image assets and stuff in this folder too

https://github.com/pact-foundation/pact-plugins/tree/main/docs